test/tests/operations/MS.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * CharEnc tests.
+ *
+ * @author tlwr [toby@toby.codes]
+ * @copyright Crown Copyright 2017
+ * @license Apache-2.0
+ */
+import TestRegister from "../../TestRegister.js";
+
+TestRegister.addTests([
+    {
+        name: "Micrsoft Script Decoder",
+        input: "##@~^DgAAAA==\\ko$K6,JCV^GJqAQAAA==^#~@",
+        expectedOutput: "MsgBox \"Hello\"",
+        recipeConfig: [
+            {
+                "op": "Micrsoft Script Decoder",
+                "args": []
+            },
+        ],
+    },
+]);
```

Comment:
Add a test to confirm that this works for JScript.Encode as well as VBE.

Reply:
I'm have a bit of hard time finding a good sample. Using the VBS script from https://gallery.technet.microsoft.com/scriptcenter/16439c02-3296-4ec8-9134-6eb6fb599880 I was able check both JS and VBS and noticed that they produce the same file output given the same file input even when you specify a different script type in the EncodeScriptFile function. I did create a more complex test though.